docs/guide/faq/README.md

```diff
@@ -85,9 +85,10 @@ You need to re-pair all you devices when:
 
 - Changing the network key (`network_key`) or panID (`pan_id`) in `configuration.yaml`.
 - Changing the Zigbee channel (`channel`) in the `configuration.yaml` might require repairing **some** devices, read the [documentation for more info](../configuration/zigbee-network.md#changing-the-zigbee-channel).
+- Changing from zStack 1.x to 3.x.
```

Comment:
I think this only happens with the CC2531 (legacy adapter), it simply doesn't have enough memory available to keep all the childs (3.x is NOT recommended for the CC2531)

Reply:
That is correct. That is exactly what I'd like to have added to the README if possible, to make it clear for people like me. In addition, is it correct only routing devices need to be re-paired, as end nodes are less strict and will keep connected? I should have added that too perhaps (I didn't do the migration yet but am planning to, might add that information from experience by then).

Reply:
Then I would make it a bit more clear by

> - Changing from zStack 1.x to 3.x. (only applies to CC2530 and CC2531)

What happens is that zstack 3 has smaller routing tables (due to the stack being larger), I guess this is what causes the disconnects.

Reply:
Yeah that would be perfect. Thanks for your review and comments!

Reply:
Updated it, thanks!